CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [16.0.0](https://github.com/Yelp/bento/compare/v15.8.0...v16.0.0) (2020-05-07)
+## [16.0.1]([c459cdd](https://github.com/pumpkinjnn/bento/commit/e36bfd25733abb35bbb74fd7fbbd3dce8e714872)) (2020-12-10)
+
+### Refactor
+* Dropped jetifier
 
+## [16.0.0](https://github.com/Yelp/bento/compare/v15.8.0...v16.0.0) (2020-05-07)
```

Comment:
This file is updated automatically by `standard-version`, you shouldn't need to do anything.